test/cmd.js

```diff
@@ -120,6 +121,111 @@ describe('express(1)', function () {
+        dir = newDir;
+        done();
+      });
+    });
+
+    mocha.after(function (done) {
+      this.timeout(30000);
+      cleanup(dir, done);
+    });
+
+    it('should create basic app', function (done) {
+      run(dir, ['--es6'], function (err, stdout) {
+        if (err) return done(err);
+        files = parseCreatedFiles(stdout, dir);
+        output = stdout;
+        assert.equal(files.length, 17);
+        done();
+      });
+    });
+
+    it('should provide debug instructions', function () {
+      assert.ok(/DEBUG=app-(?:[0-9\.]+):\* (?:\& )?npm start/.test(output));
+    });
+
+    it('should have basic files', function () {
+      assert.notEqual(files.indexOf('bin/www'), -1);
+      assert.notEqual(files.indexOf('app.js'), -1);
+      assert.notEqual(files.indexOf('package.json'), -1);
+    });
+
+    it('should have jade templates', function () {
+      assert.notEqual(files.indexOf('views/error.jade'), -1);
+      assert.notEqual(files.indexOf('views/index.jade'), -1);
+      assert.notEqual(files.indexOf('views/layout.jade'), -1);
+    });
+
+    it('should have a package.json file', function () {
+      var file = path.resolve(dir, 'package.json');
+      var contents = fs.readFileSync(file, 'utf8');
+      assert.equal(contents, '{\n'
+        + '  "name": ' + JSON.stringify(path.basename(dir)) + ',\n'
+        + '  "version": "0.0.0",\n'
+        + '  "private": true,\n'
+        + '  "scripts": {\n'
+        + '    "start": "node ./bin/www"\n'
+        + '  },\n'
+        + '  "dependencies": {\n'
+        + '    "body-parser": "~1.13.2",\n'
+        + '    "cookie-parser": "~1.3.5",\n'
+        + '    "debug": "~2.2.0",\n'
+        + '    "express": "~4.13.1",\n'
+        + '    "jade": "~1.11.0",\n'
+        + '    "morgan": "~1.6.1",\n'
+        + '    "serve-favicon": "~2.3.0"\n'
+        + '  },\n'
+        + '  "babel": {\n'
+        + '    "presets": [\n'
+        + '      "es2015"\n'
+        + '    ]\n'
+        + '  },\n'
+        + '  "devDependencies": {\n'
+        + '    "babel-core": "^6.1.2",\n'
+        + '    "babel-preset-es2015": "^6.1.2"\n'
+        + '  }\n'
+        + '}');
+    });
+
+    it('should have installable dependencies', function (done) {
+      this.timeout(120000);
+      npmInstall(dir, done);
+    });
+
+    it('should export an express app from app.js', function () {
+      this.timeout(30000);
+      var file = path.resolve(dir, 'app.js');
+      var app = require(file).default;
+      assert.equal(typeof app, 'function');
+      assert.equal(typeof app.handle, 'function');
+    });
+
+    it('should respond to HTTP request', function (done) {
+      var file = path.resolve(dir, 'app.js');
+      var app = require(file).default;
+      request(app)
+      .get('/')
+      .expect(200, /<title>Express<\/title>/, done);
+    });
+
+    it('should generate a 404', function (done) {
+      var file = path.resolve(dir, 'app.js');
+      var app = require(file).default;
```

Comment:
Why can it not be `require(file)`? Seems like the ES6 app is subtly different from the others and the rest of Node.js things if it falls under a `default` property.